python-checks/src/main/resources/org/sonar/l10n/py/rules/python/LongIntegerWithLowercaseSuffixUsage.html

```diff
@@ -1,17 +1,14 @@
-<p>
-Although both lower case 'l' and upper case 'L' are allowed as suffix for long integers, it is strongly recommended to always use 'L', since the letter 'l' looks too much like the digit '1'.
-</p>
-
-<p>
-The following code snippet illustrates this rule :
-</p>
-
+<p>The long suffix should always be written in uppercase, i.e. 'L', as the lowercase 'l' can easily be confused with the digit one '1'.</p>
+<h2>Noncompliant Code Example</h2>
+<pre>
+long n = 10l;  // Noncompliant; easily confused with one zero one
+</pre>
+<h2>Compliant Solution</h2>
 <pre>
-return 1l  # Non-Compliant
-return 1L  # Compliant
+long n = 10L;
 </pre>
+<h2>See Also</h2>
+<ul>
+  <li> {rule:python:S818} - Literal suffixes should be upper case </li>
```

Comment:
@pynicolas Whole "See also" section removed from RSPEC. I didn't create any ticket to implement S818, as not everything in S818 is relevant for Pyton. For example, hex format can be like '02x' or '02X' for good reasons (to generate "ff" vs. "FF" for 255).